### PR TITLE
Create ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by tba

Ticket: [MTM-60605](https://cumulocity.atlassian.net/browse/MTM-60605)
Original PR: [6979](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6979)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: add support for module less plugins (#6979)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Support module-less plugins
```

File: content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60605
+version: 1020.24.0
+---
+add support for module less plugins (#6979)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md:17]: The Claude AI suggested to use the following body:
```suggestion
To simplify plugin development and deployment, support for module-less plugins has been added. With this change, plugins no longer require a module to function, allowing them to be implemented and deployed more easily. This impacts plugin developers who can now create plugins without the overhead of a module, and users who can benefit from a wider variety of available plugins that are simpler to install and use.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md (hunk 1 above)
Comment (MWindel) [reply content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md:3]: Plugins without models are now supported

Comment (reey) [reply content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md:3]:
```suggestion
title: Plugins without modules are now supported
```

**Review by reey — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-60605
+version: 1020.24.0
+---
+To simplify plugin development and deployment, support for module-less plugins has been added. With this change, plugins no longer require a module to function, allowing them to be implemented and deployed more easily. This impacts plugin developers who can now create plugins without the overhead of a module, and users who can benefit from a wider variety of available plugins that are simpler to install and use.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-24-0-add-support-for-module-less-plugins-6979.md:17]:
```suggestion
To simplify plugin development and deployment, support for module-less plugins has been added. With this change, plugins no longer require a module to function, allowing to implement and deploy plugins more easily. This impacts plugin developers who can now create plugins without the overhead of a module, and users who can benefit from a wider variety of available plugins that are simpler to install and use.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**